web/annotation_layer_builder.css

```diff
@@ -97,6 +99,66 @@
   width: 115%;
 }
 
+.annotationLayer .buttonWidgetAnnotation.checkBox label,
+.annotationLayer .buttonWidgetAnnotation.radioButton label {
```

Comment:
Unless I'm mistaken, most of these rules are the same as those used above in https://github.com/mozilla/pdf.js/pull/7898/files#diff-6ebc03110e3e120aeeedb42bf7ba8d90R44, hence moving these lines there (with possible necessary overrides here) might make the code more readable/maintainable.